app/helpers/results_helper.rb

```diff
@@ -6,22 +6,36 @@ module ResultsHelper
   def render_field_item(doc)
     (doc[:document].has_highlight_field? doc[:field]) ?
       doc[:document].highlight_field(doc[:field]).join(", ").html_safe :
-        doc[:document][doc[:field]].join(", ").html_safe
+        doc[:document][doc[:field]].join(", ").truncate(450).html_safe
```

Comment:
@eshadatta Once you rebase you'll see all the highlighting stuff has been removed. And this function can be a one-liner